src/logic/wallets/pairing/module.ts

```diff
@@ -36,11 +37,21 @@ const getClientMeta = (): IClientMeta => {
 
 const createPairingProvider = (): WalletConnectProvider => {
   const STORAGE_ID = 'SAFE__pairingProvider'
+
+  const rpc = getChains().reduce((map, { chainId, rpcUri }) => {
+    return {
+      ...map,
+      [parseInt(chainId, 10)]: getRpcServiceUrl(rpcUri),
+    }
+  }, {})
```

Comment:
Can we extract this into a function again since its being used in two places. Previously it was called `getRpcMap`. The implementation in `walletList.ts` also slightly differs.

Reply:
Here it requires numerical keys, whereas onboard requires strings.